.github/workflows/test-e2e.yaml

```diff
@@ -45,7 +45,7 @@ jobs:
         with:
           repository: vechain/thor-e2e-tests
           # https://github.com/vechain/thor-e2e-tests/tree/00bd3f1b949b05da94e82686e0089a11a136c34c
```

Comment:
```suggestion
          # https://github.com/vechain/thor-e2e-tests/tree/5fb22b0b3f3d28d5083e2d78e27e711da0425cdc
```

Reply:
Ignoring as I pushed updated the sha again and fix the comment too